spec/components/bib_item_component_spec.rb

```diff
@@ -8,14 +8,37 @@
   it 'renders a bib item with an author' do
     hold.record['fields']['bib']['fields']['title'] = 'A wonderful title'
     hold.record['fields']['bib']['fields']['author'] = 'A wonderful author'
+
     result = render_inline(described_class, bibitem: hold)
-    expect(result.to_html).to include 'A wonderful title / A wonderful author</a>'
+    expect(result.to_html).to include 'A wonderful title / A wonderful author'
   end
 
   it 'renders a bib item without an author' do
     hold.record['fields']['bib']['fields']['title'] = 'A wonderful title'
     hold.record['fields']['bib']['fields']['author'] = ''
     result = render_inline(described_class, bibitem: hold)
-    expect(result.to_html).to include 'A wonderful title</a>'
+    expect(result.to_html).to include 'A wonderful title'
+  end
+
+  context 'when bib item type is anything but PALCI' do
+    before do
+      hold.record['fields']['item']['fields']['itemType']['key'] = 'ATLAS'
+    end
+
+    it 'renders an linked title' do
```

Comment:
```suggestion
    it 'renders a linked title' do
```
sorry for the totally unnecessary suggestion but I cannot help it :)